lib/counter_web/live/counter_live.ex

```diff
@@ -0,0 +1,28 @@
+defmodule CounterWeb.CounterLive do
+  use Phoenix.LiveView
+
+  def mount(_session, _, socket) do
```

Comment:
@JuanMaGiraldo please explain how a mount function works

Reply:
`mount/3` is a callback that is invoked when LiveView is first rendered, it's like a constructor for the view.
`mount/3` is invoked with the current params, session and the socket, in this case, I have wrongly positioned the section parameter.